assets/js/app.js

```diff
@@ -0,0 +1,9 @@
+$('.start').html(AppTemplates.start(heroes));
+
+$('.start').click(function() {
+  html = ('.battle');
+});
+
+$(game).html(AppTemplates.start());
+$(game).html(AppTemplates.battle());
```

Comment:
Don't show the `battle` screen until someone has clicked on something in the start screen